README.md

```diff
@@ -11,33 +11,72 @@ Registrator is a GitHub app that automates creation of registration pull request
 
 [![install](https://img.shields.io/badge/-install%20app-blue.svg)](https://github.com/apps/juliaregistrar/installations/new)
 
+#### Permissions and subscribed events for the app
```

Comment:
It's kind of unclear to me without additional context what this refers to.

Reply:
Those are instructions for someone who wants to host Registrator. There are 2 kinds of documentation here: 

1) For someone who has installed Registrator on their repo and wants to know how to register a new version.

2) For someone who wants to host Registrator and wants to know about the GitHub settings for the app. What to do for private packages etc.

I think 1) should go to README.md and 2) Should be in a file called docs.md (which is full_readme.md right now but I never got around to cleaning up/updating that file).

Reply:
Okay, I've removed this right now. Will add it later in a different PR.